src/Components/OptionSelector.js

```diff
@@ -0,0 +1,26 @@
+import { Component } from 'react'
+import Form from 'react-bootstrap/Form'
+
+class OptionSelector extends Component {
+
+
+helperFunction = (e) => {
+
+  const numOfHorns = e.target.value;
+  this.props.handleSelect(numOfHorns)
+
+}
```

Comment:
Really nice work Ediberto!

Feedback here: If you have time to go back in here, I would try and rename this method. This is an event handler so instead the name can start with `handle` just like you did for `handleSelect` in `Main`.